eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -12,7 +14,7 @@
   <Target Name="UpdateRuntimePack"
           AfterTargets="ResolveFrameworkReferences">
     <ItemGroup>
-      <ResolvedRuntimePack Update="@(ResolvedRuntimePack)" PackageDirectory="$(RuntimePackDir)" />
+      <ResolvedRuntimePack Update="@(ResolvedRuntimePack)" PackageDirectory="$(MicrosoftNetCoreAppRuntimePackRidDir)/../../" />
```

Comment:
Should this be `$(MicrosoftNetCoreAppRuntimePackDir)` instead?

Reply:
If I do that, then I need to pass in the two properties (MicrosoftNetCoreAppRuntimePackRidDir and MicrosoftNetCoreAppRuntimePackDir) since the one that the Wasm targets read is the Rid-specific one.

Reply:
which I'm fine if we want to do that, but I thought the idea was to avoid duplication of properties.

Reply:
It just feels like there should be one base property that needs to be set, and everything flows from there. If not, then I'm fine with what you have now.